packages/mobile/src/navigator/DrawerNavigator.tsx

```diff
@@ -157,9 +175,19 @@ function CustomDrawerContent(props: DrawerContentComponentProps<DrawerContentOpt
         <Text style={fontStyles.label}>Account No.</Text>
         <View style={styles.accountOuterContainer}>
           <View style={styles.accountInnerContainer}>
-            <AccountNumber address={account || ''} />
+            <AccountNumber address={account || ''} location={Screens.DrawerNavigator} />
           </View>
         </View>
+        {devModeActive && (
+          <>
+            <Text style={fontStyles.label}>Session ID</Text>
+            <View style={styles.accountOuterContainer}>
+              <View style={styles.accountInnerContainer}>
+                <SessionId sessionId={storedSessionId || ''} />
+              </View>
+            </View>
+          </>
+        )}
```

Comment:
Feels like this should go in Settings with the other devModeActive-only settings and values